packages/core/src/server/middlewares/settings-and-store.ts

```diff
@@ -22,17 +22,23 @@ export const settingsAndStore = (packages) => async (
     : nameOption;
 
   // Get settings.
-  ctx.state.settings = await getSettings({
+  ctx.frontity.settings = await getSettings({
     url: ctx.href,
     name,
   });
 
   // Create the store.
-  ctx.state.store = createStore({
-    settings: ctx.state.settings,
+  ctx.frontity.store = createStore({
+    settings: ctx.frontity.settings,
     packages,
     url: ctx.URL,
   });
 
+  // Populate the context for custom middleware.
+  Object.assign(ctx, {
+    ctx: ctx,
+    ...ctx.frontity.store,
+  });
```

Comment:
There's no need for destructuring here 🙂

```suggestion
  Object.assign(ctx, { ctx }, ctx.frontity.store);
```

Reply:
Hum, I didn't know it took more than one value